engines/gob/sound/sounddesc.cpp

```diff
@@ -161,18 +161,21 @@ bool SoundDesc::loadWAV(byte *data, uint32 dSize) {
 
 	int wavSize, wavRate;
 	byte wavFlags;
-	uint16 wavtype;
+	uint16 wavType;
+	int numChannels; 
 
-	if (!Audio::loadWAVFromStream(stream, wavSize, wavRate, wavFlags, &wavtype, nullptr))
+	if (!Audio::loadWAVFromStream(stream, wavSize, wavRate, wavFlags, &wavType, &numChannels))
 		return false;
 
 	if (wavFlags & Audio::FLAG_16BITS) {
 		_mixerFlags |= Audio::FLAG_16BITS;
 		wavSize >>= 1;
 	}
 
-	if (wavFlags & Audio::FLAG_STEREO) {
-		warning("TODO: SoundDesc::loadWAV() - stereo");
+	if (numChannels == 2) {
+		_mixerFlags |= Audio::FLAG_STEREO;
+	} else if (numChannels != 1) {
+		warning("SoundDesc::loadWAV unsupported number of channels %d", numChannels);
```

Comment:
The number of channels is [already handled internally in `loadWAVFromStream()`](https://github.com/scummvm/scummvm/blob/c994c5ee8afad7e6ae299fb3d615410b6a441073/audio/decoders/wave.cpp#L201) - please use the original code:

```suggestion
	if (wavFlags & Audio::FLAG_STEREO)
		_mixerFlags |= Audio::FLAG_STEREO;
```